packages/bcer-api/app/src/notification/entities/notification.entity.ts

```diff
@@ -0,0 +1,74 @@
+import { Column, CreateDateColumn, Entity, PrimaryGeneratedColumn, UpdateDateColumn } from "typeorm";
+import { NotificationRO } from "../ro/notification.ro";
+
+@Entity('notification')
+export class NotificationEntity {
+    @PrimaryGeneratedColumn('uuid')
+    id: string;
+
+    @Column('varchar', {
+        length: 255,
+        nullable: true
+    })
+    sender: string;
+
+    @Column('varchar', {
+        length: 64,
+        nullable: true,
+    })
+    title: string;
+
+    @Column('varchar', {
+        length: 650,
+        nullable: false,
+    })
+    message: string;
+
+    @CreateDateColumn({
+        name: 'created_at'
+    })
+    createdAt: Date
+
+    @UpdateDateColumn({
+        name: 'updated_at'
+    })
+    updatedAt: Date;
+
+    @Column('boolean', {
+        nullable: false,
+        default: false
+    })
+    completed: boolean;
+
+    @Column('int', {
+        nullable: true,
+    })
+    success: number;
+
+    @Column('int', {
+        nullable: true,
+    })
+    fail: number; 
+
+    @Column('json', {
```

Comment:
Just want to confirm that a `json` field doesn't have any max-length defaults applied, as it could contain a fair bit of data if lots of messages fail